### PR TITLE
[RA1 Ch02] Update Network Resiliency Requirement

### LINKED ISSUE #2061: [RA1 Ch02] Network Resiliency Reqt -- make specific
req.inf.ntw.07 | Network | The Architecture must support network resiliency. | RA-1 3.4.2.2. "Network"
-- | -- | -- | --

Need to be more specific

### PR DESCRIPTION
**Pull request by pgoyal01**
Fixes #2061 
Clarify req.inf.ntw.07

### CODE REVIEW

File: doc/ref_arch/openstack/chapters/chapter02.md [hunk 1]
```diff
@@ -352,7 +352,7 @@ The Platform is assumed to provide configurable alerting and notification capabi
...
 | `req.inf.ntw.03` | Network | The Architecture **must** support low latency and high throughput traffic needs. | [RA-1 4.2.3. "Network Fabric"](./chapter04.md#423-network-fabric) |
 | `req.inf.ntw.05` | Network | The Architecture **must** allow for East/West tenant traffic within the cloud (via tunnelled encapsulation overlay such as VXLAN or Geneve). | [RA-1 4.2.3. "Network Fabric"](./chapter04.md#423-network-fabric) |
-| `req.inf.ntw.07` | Network | The Architecture **must** support network resiliency. | [RA-1 3.4.2.2. "Network"](./chapter03.md#3422-network) |
+| `req.inf.ntw.07` | Network | The Architecture **must** support network resiliency by maintaining an acceptable level of service in the face of faults and challenges to normal operation (from wikipedia). | [RA-1 3.4.2.2. "Network"](./chapter03.md#3422-network) |
```
Comment (CsatariGergely) [inline doc/ref_arch/openstack/chapters/chapter02.md:355]:
```suggestion
| `req.inf.ntw.07` | Network | The Architecture **must** support network resiliency by [maintaining an acceptable level of service in the face of faults and challenges to normal operation](https://en.wikipedia.org/wiki/Resilience_(network)). | [RA-1 3.4.2.2. "Network"](./chapter03.md#3422-network) |
```

**Review by CsatariGergely — CHANGES_REQUESTED** [review]

**Comment by pgoyal01** [comment]
@CsatariGergely Thanks. Accepted suggestion. Please review.

**Comment by pgoyal01** [comment]
Made changes as per suggestion by @petorre - Thanks. Replaced the added wording and wikipedia url with pointer to Glossary.

**Review by petorre — APPROVED** [review]

**Review by ulikleber — APPROVED** [review]

**Comment by pgoyal01** [comment]
@CsatariGergely Can you please re-review? Thanks

**Review by sukhdevkapur — APPROVED** [review]

**Review by CsatariGergely — APPROVED** [review]

**Review by karinesevilla — APPROVED** [review]

**Pull request merged**